### ISSUE TITLE
Syntax error in elasticsearch_plugins documentation

### ISSUE THREAD

**Issue by hatdropper1977** [elastic/rally#389]
**Rally version** (get with `esrally --version`):
```bash
(rally)[centos@ip-10-0-0-48 ~]$ esrally --version
esrally 0.8.1
```

**Invoked command**:
```bash
(rally)[centos@ip-10-0-0-48 ~]$ esrally --track=pmc --revision=latest --distribution-version=6.1.1 --user-tag="intention:Rally-Downloads-Server-Baseline" --elasticsearch-plugins=xpack:security --client-options="use_ssl:true,verify_certs:false,basic_auth_user:'rally',basic_auth_password:'rally-password'" --cluster-health=yellow

...

[ERROR] Cannot race. ('Error on host 127.0.0.1', Unknown plugin [xpack]. List the available plugins with esrally list elasticsearch-plugins --distribution-version=VERSION.)
```

**Configuration file (located in `~/.rally/rally.ini`))**:
```yaml
[runtime]
java.home = /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.151-5.b12.el7_4.x86_64

[benchmarks]
local.dataset.cache = ${node:root.dir}/data

[reporting]
datastore.type = elasticsearch
datastore.host = ip-10-0-0-10.ec2.internal
datastore.port = 9200
datastore.secure = True
datastore.user = elastic
datastore.password = <my password here>

[tracks]
default.url = https://github.com/elastic/rally-tracks

[teams]
default.url = https://github.com/elastic/rally-teams

[defaults]
preserve_benchmark_candidate = False

[distributions]
release.1.url = https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-{{VERSION}}.tar.gz
release.2.url = https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/{{VERSION}}/elasticsearch-{{VERSION}}.tar.gz
release.url = https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{{VERSION}}.tar.gz
release.cache = true
```

**JVM version**:
```bash
(rally)[centos@ip-10-0-0-48 ~]$ java -version
openjdk version "1.8.0_151"
OpenJDK Runtime Environment (build 1.8.0_151-b12)
OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)
```

**OS version**:
```bash
(rally)[centos@ip-10-0-0-48 ~]$ uname -a
Linux ip-10-0-0-48.ec2.internal 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 20 20:32:50 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

**Description of the problem including expected versus actual behavior**:
I executed the command mentioned in http://esrally.readthedocs.io/en/stable/elasticsearch_plugins.html
I got an error, since I typed xpack instead of x-pack

**Steps to reproduce**:
Execute the command above.

**Provide logs (if relevant)**:
```bash
See error in command execution above.
```

**Describe the feature**:
Update the docs.

**Comment by danielmitterdorfer** [comment]
You are right. Thanks for spotting it and also providing a PR.

**Comment by danielmitterdorfer** [comment]
Closed by #390
